src/services/phoneAuth.ts

```
@@ -0,0 +1,117 @@
+    console.error('휴대폰 인증 실패:', error);
+    return {
+      success: false,
+      message: '휴대폰 인증에 실패했습니다.',
+      isVerified: false,
+    };
+  }
+};
+
+// 실제 SMS 전송 함수 (백엔드 API 사용)
+export const sendSmsReal = async (
+  client: ApolloClient<any>, 
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)